server/websocket.go

```diff
@@ -30,6 +30,7 @@ const (
 	wsEventCallEnd          = "call_end"
 	wsEventUserRaiseHand    = "user_raise_hand"
 	wsEventUserUnraiseHand  = "user_unraise_hand"
+	wsEventUserReact        = "user_reaction"
```

Comment:
```suggestion
	wsEventUserReacted        = "user_reacted"
```

Reply:
Is it ok if we keep it consistent and use `wsEventUserReact = "user_react"`?

Reply:
`react` would be the client action and `reacted` the event that gets broadcast. But yeah, it's not like we're being consistent with the others so whatever makes sense is fine.